lang/nl/route.php

```diff
@@ -9,7 +9,9 @@
     "update_password" => "wachtwoord-bijwerken",
     "update" => "bijwerken",
     "order" => "bestelling",
-    "overview" => "overzicht",
+    "overview_admin" => "overzicht",
+    "overview_user" => "overzicht-bestellingen",
```

Comment:
Je kan gewoon "overview" doen, en die in de route prefix gebruiken. Dan heb je:
http://127.0.0.1:8000/producten/overzicht/Kabouters
en
http://127.0.0.1:8000/bestelling/overzicht
Ook is vgm dat overview niet alleen voor de admin, maar voor het overzicht van producten